configs/automated_screens/intermetallics_CO2RR.yml

```diff
@@ -1,16 +1,12 @@
-dask:
-  partitions: -1 # Use automatic guess for number of partitions
-
 memory_cache_location: '/home/jovyan/shared-scratch/catlas/cache_github'
```

Comment:
I am pretty sure this file is practically deprecated. Maybe we should get rid of it?